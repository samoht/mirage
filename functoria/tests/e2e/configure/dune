(executable (name test) (libraries astring f0))

(alias
  (name runtest)
  (deps (package functoria) (source_tree src))
  (action (progn
    (run %{exe:test.exe})
    (diff? src/main.ml.expected src/main.ml)
    (diff? src/dune.expected src/dune)
    (diff? src/dune.build.expected src/dune.build)
    (diff? src/dune.config.expected src/dune.config)
    (diff? src/key_gen.ml.expected src/key_gen.ml))))
