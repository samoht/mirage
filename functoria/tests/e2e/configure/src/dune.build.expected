(executable
   (name      main)
   (modules (:standard \ config))
   (libraries cmdliner fmt functoria-runtime))
